docs/HttpErrorResponses.md

```diff
@@ -0,0 +1,130 @@
+# Helium HTTP/400 Error Response Design
+
+This design guide covers the types of responses and format used when an HTTP/400 error occurs. Aligning with [RFC 7807](https://tools.ietf.org/html/rfc7807), the error responses adhere to this specification with a additional extension members containing a collection of parameter validation errors.
+
+## Scope
+
+The HTTP/400 error responses outlined in this document currently only cover API input parameter validation at this time.
+
+## Base Problem Details Object
+
+The HTTP response contains a "Problem Details Object" which has the following members:
+
+- "**type**" (string): An optional URI which identifies the problem type.
+- "**title**" (string): Summary of the problem.
+- "**status**" (number): Currently scoped to HTTP/400
+- "**detail**" (string): A message specific to this type of problem.
+- "**instance**" (string): A URI reference indicating the HTTP route referenced by the API (excluding query string parameters)
+
+Note: Object order is not pre-defined or set and can vary.
+
+## Extension Members
+
+To provide more specific information about the error condition, the RFC permits extension members to be defined with no limit to the schema. As such, an object array called `validationErrors` is used to hold one or more objects with the following members:
+
+- "**code**" (string): The error type specific to the validation issue in the collection
+- "**target**" (string): The name of the parameter which did not validate correctly
+- "**message**" (string): A descriptive message outlining the constrains of the input
+
+## Error Types
+
+Given the above extension member called `validationErrors`, the `code` property can hold the following values:
+
+1. **NullValue**: Used when the input parameter is null
+2. **InvalidValue**: Used when the input parameter is not the correct type or outside the bounds of the required value(s)
+
+## Application Type
+
+In accordance with [RFC 7807](https://tools.ietf.org/html/rfc7807), the application type, also known as the `Content-Type` property of the HTTP response header, shall be set to: `application/problem+json`
+
+## Parameter Response Messages
+
```

Comment:
standardize how some responses end with period and others don't

Reply:
Fixed in PR update